doozer/doozerlib/cli/scan_osh.py

```diff
@@ -0,0 +1,157 @@
+        self.dry_run = dry_run
+        self.brew_tags = []
+        self.specific_nvrs = nvrs
+        self.check_triggered = check_triggered
+        self.nvrs = []
+
+        # Initialize runtime and brewhub session
+        self.runtime.initialize(clone_distgits=False)
+        self.koji_session = koji.ClientSession(self.runtime.group_config.urls.brewhub)
+
+    async def untriggered_nvrs(self):
+        """
+        So we might have already triggered a scan for the same NVR. If this flag is enabled, we want to check
+        if it has and not re-trigger it again.
+        But please note that this will take a lot of time since it has to run for all 200+ packages.
+        """
+        untriggered_nvrs = []
+        for nvr in self.nvrs:
+            rc, _, _ = cmd_gather(f"osh-cli find-tasks --nvr {nvr}")
+
+            if rc == 1:
+                # Get the list of NVRS that a task hasn't already been triggered for
+                untriggered_nvrs.append(nvr)
+
+        self.nvrs = untriggered_nvrs
+
+    def get_tagged_latest(self, tag):
+        """
+        Returns the latest RPMs and builds tagged in to the candidate tag received as input
+        """
+        latest_tagged = self.koji_session.listTagged(tag=tag, latest=True)
+        if latest_tagged:
+            return latest_tagged
+        else:
+            return []
+
+    def get_tagged_all(self, tag):
+        """
+        Returns all the RPMs and builds that are currently in the candidate tag received as input
+        """
+        latest_tagged = self.koji_session.listTagged(tag=tag, latest=False)
+        if latest_tagged:
+            return latest_tagged
+        else:
+            return []
+
+    async def trigger_scan(self):
+        cmd_template = "osh-cli mock-build --config={config} --brew-build {nvr} --nowait"
+        for nvr in self.nvrs:
+            if "container" in nvr:
+                cmd = cmd_template.format(config="cspodman", nvr=nvr)
+
+            else:
+                if "el7" in nvr:
+                    rhel_version = 7
+                elif "el8" in nvr:
+                    rhel_version = 8
+                elif "el9" in nvr or nvr.startswith("rhcos"):
+                    rhel_version = 9
+                else:
+                    self.runtime.logger.error("Invalid RHEL version")
+                    return
+
+                cmd = cmd_template.format(config=f"rhel-{rhel_version}-x86_64", nvr=nvr)
+
+            message = f"Ran command: {cmd}"
+
+            if not self.dry_run:
+                fire_and_forget(self.runtime.cwd, cmd)
+            else:
+                message = "[DRY RUN] " + message
+
+            self.runtime.logger.info(message)
+
+    async def run(self):
+        if not self.specific_nvrs:
+            tags = self.runtime.get_errata_config()["brew_tag_product_version_mapping"].keys()
+            for tag in tags:
+                major, minor = self.runtime.group.replace("openshift-", "").split(".")
```

Comment:
there's a method called `get_major_minor` that we can use here

Reply:
ahh! Thank you! https://github.com/openshift-eng/art-tools/pull/51/commits/7c715b307aa02efa5d521ca0e931c9a2ae18766e